packages/angular-virtual/e2e/app/src/app/app.routes.ts

```
@@ -0,0 +1,29 @@
+import type { Routes } from '@angular/router'
```

ESLint

[error] `@angular/router` type import should occur after import of `./stale-index.component`

(import/order)